datadog_checks_downloader/datadog_checks/downloader/download.py

```diff
@@ -72,6 +79,13 @@ def __init__(
         self.__root_layout_type = root_layout_type
         self.__root_layout = ROOT_LAYOUTS[self.__root_layout_type]
 
+        self.__disable_verification = disable_verification
+
+        if self.__disable_verification:
```

Comment:
One more thing, please: I think we should not only log the warning, but also interactively prompt the user to confirm this action. IMHO, we should not make it easy to automate to override security (just like browsers make it nontrivial now to skip TLS).

Reply:
Sounds reasonable. At the same time, since there is no real urgent need for this PR yet, I suggest holding off on it until we can be sure that it would indeed reasonably solve previous incidents that motivated it, which is something we should discuss offline.